ledger/participant-integration-api/src/main/scala/platform/store/backend/postgresql/PGField.scala

```diff
@@ -3,33 +3,8 @@
 
 package com.daml.platform.store.backend.postgresql
 
-import java.time.{Instant, ZoneOffset}
-import java.time.format.DateTimeFormatter
-
 import com.daml.platform.store.backend.common.Field
 
-private[postgresql] trait PGTimestampBase[FROM, TO] extends Field[FROM, TO, String] {
```

Comment:
Nice side effect of this change: less differences between StorageBackend implementations.

Reply:
yaaaaaay!